README.md

```diff
@@ -69,3 +69,11 @@ Combination of `cloudfront:setup` and `cloudfront:build:deploy` commands with or
 ### `serverless:remove`
 
 Remove an entire stack configured in `serverless.yml` via CloudFormation.
+
+### S3 bucket URL
+
+[http://shop-react-redux-cloudfront2.s3-website-eu-west-1.amazonaws.com/](http://shop-react-redux-cloudfront2.s3-website-eu-west-1.amazonaws.com/)
```

Comment:
hey @JavokhirbekMirzaakbarov ! this endpoint shows 400 Error but it should show "403 Access Denied error" according to the task. You probably need to define OriginAccessIdentity in server less config

Reply:
I have fixed the error with endpoint please review!